elasticsearch-rails/lib/elasticsearch/rails/lograge.rb

```diff
@@ -33,8 +33,14 @@ class Railtie < ::Rails::Railtie
             include Elasticsearch::Rails::Instrumentation::ControllerRuntime
           end
 
+          previous_lograge_custom_options = config.lograge.custom_options
           config.lograge.custom_options = lambda do |event|
-            { es: event.payload[:elasticsearch_runtime].to_f.round(2) }
+            previous_lograge_custom_options_hash = if previous_lograge_custom_options
+              previous_lograge_custom_options.call(event)
```

Comment:
Good catch @halilim. This assumes that `previous_lograge_custom_options` responds to `call`, which it won't when it's a Hash.